.rubocop.yml

```diff
@@ -0,0 +1,8 @@
+Style/Documentation:
```

Comment:
@Robgra13 Best to reuse some existing template to have consistency across projects: https://github.com/BookingSync/rails-transactional-outbox/blob/master/.rubocop.yml

Reply:
Just remember to remove the excluded files which don't exist in this gem ;)